src/main/webapp/form.html

```diff
@@ -11,7 +11,7 @@
   <body>
     <div class="content">
       <div class="container">
-        <h1>StudentRecs</h1>
+        <h1>Student Recs</h1>
```

Comment:
Shall we use `Recommendations` here instead of `Recs` for consistency with the title above?